### ISSUE TITLE
Issue reproducing the uploaded pretrained model

### ISSUE THREAD

**Issue by unofgithub** [lschmidtke/shape_templates#8]
Hello!

I have been trying to reproduce the pretrained checkpoint using the uploaded code (model.py, networks.py, the utils, train.py) to train on human3.6m, but it is not successful. Could you please let me know if you could upload the exact code repository that was used to get the uploaded pretrained checkpoint? Thank you so much in advance!

**Comment by unofgithub** [comment]
Also, could you please let me know approximately what the size of the data you trained on was (i.e. approximately how many pairs of images the pretrained model was trained on)? I tried training the model on around 180,000 pairs, but was not able to reproduce the pretrained checkpoint.

**Comment by unofgithub** [comment]
Hello,

Specifically, these are the results I am getting:
<img width="333" alt="image" src="https://user-images.githubusercontent.com/31677778/222877420-a23b4b33-1aa7-4d6b-bd0e-4460faf92dc3.png">

Could you please let me know how I might fix this, since it does not match the pretrained model? Thank you so much!

**Comment by lschmidtke** [comment]
Hi! The code should produce the same results of the provided checkpoint. How do the reconstructed images look like?
The model was trained on roughly 600k images.

**Comment by unofgithub** [comment]
The reconstructed images look like:
![image](https://user-images.githubusercontent.com/31677778/222983605-ab94d155-5be9-4f95-ac39-ed49ed11e584.png)

**Comment by lschmidtke** [comment]
Okay, the reconstructions don't look good. What's your batch size and learning rate? Also, how many epochs are you training for?

**Comment by unofgithub** [comment]
Hello! My batch size is 48. Learning rate is 0.0001. I'm training 50 epochs.

**Comment by lschmidtke** [comment]
That seems reasonable to me. I will run the code myself with those parameters and I'll get back to you.

**Comment by unofgithub** [comment]
Hello,

Could you please let me know when there are updates? Thank you so much!

**Comment by lschmidtke** [comment]
Hi,

I encountered the same issue, I suppose it's due to a newer version of one of the packages. I will try and find the cause.
